tests/python/integration_test.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) 2018 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Provides a base class for integration tests.
+"""
+
+from __future__ import print_function
+
+import os
+import tempfile
+import atexit
+from mock import Mock
+
+import unittest2
+
+import sgtk
+from sgtk.util import sgre as re
+from sgtk.util.filesystem import safe_delete_folder
+import tk_toolchain.authentication
+
+
+class DesktopServerIntegrationTest(unittest2.TestCase):
```

Comment:
Any reason why we don't use the SgtkIntegrationTest class? When we wrote it it was meant to be reused, but we never had the opportunity until now.